src/constants/notifications.ts

```diff
@@ -22,6 +22,7 @@ export enum NotificationType {
   FillWithNoOrder = 'FillWithNoOrder',
   Order = 'Order',
   CosmosWalletLifecycle = 'CosmosWalletLifecycle', // lifecycle events for cosmos wallets
+  DepositNotificationLifecycle = 'DepositNotificationLifecycle', // lifecycle events for deposit notifications
```

Comment:
nit: This isn't really a `Lifecycle` event. Maybe re-name to `DepositAddressEvents`